core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -698,6 +710,43 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def listObjectsWithPrefix(bucketName: String, objectNamePrefix: String): Future[List[StorageObject]] = {
```

Comment:
This is essentially the same as the wb-libs version, right?

Reply:
yup lifted it from wb-libs. if i changed anything it was only to make it rawls-friendly